From 53ea04a0e01da94846a234ddc551b09dcef3be13 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Mon, 21 Sep 2015 18:51:21 +0800
Subject: [PATCH 0529/5242] MLK-11566-6 mlb: add i.mx6sx support

commit  19eaebcd193d7268b71cb6ab13f1ea23affc76ed from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX6SX has MLB150, add support for it.

Signed-off-by: Anson Huang <b20788@freescale.com>
(cherry picked from commit 99d374da7d49d45800821b28d55e032387f1cc67)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/mlb/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/mlb/Kconfig b/drivers/mxc/mlb/Kconfig
index 493259f..c1d902b 100644
--- a/drivers/mxc/mlb/Kconfig
+++ b/drivers/mxc/mlb/Kconfig
@@ -9,7 +9,7 @@ config MXC_MLB
 
 config MXC_MLB150
 	tristate "MLB150 support"
-	depends on SOC_IMX6Q
+	depends on (SOC_IMX6Q || SOC_IMX6SX)
 	select MXC_MLB
 	---help---
 	Say Y to get the MLB150 support.
-- 
1.7.9.5

